src/msg/async/ProtocolV2.cc

```diff
@@ -2626,6 +2628,7 @@ CtPtr ProtocolV2::reuse_connection(AsyncConnectionRef existing,
     ceph_assert(!connection->delay_state);
   }
   exproto->reset_recv_state();
+  exproto->pre_auth.enabled = false;
```

Comment:
also add `exproto->pre_auth.rxbuf.clear();` below this line

Reply:
that is already done by the reset_recv_state, right?

Reply:
you're right, disregard this comment